MidasCivil_Adapter/Convert/ToBHoM/Properties/ToProfile.cs

```diff
@@ -81,6 +100,10 @@ public static IProfile ToProfile(List<string> sectionProfile, string shape, stri
                         0, 0);
                     break;
                 case "C":
+                    if (System.Convert.ToDouble(sectionProfile[4]).LengthToSI(lengthUnit) > oM.Geometry.Tolerance.Distance || System.Convert.ToDouble(sectionProfile[5]).LengthToSI(lengthUnit) > oM.Geometry.Tolerance.Distance)
+                    {
+                        Engine.Reflection.Compute.RecordWarning("Asymmetric channel sections are not yet supported in the BHoM_Engine");
+                    }
```

Comment:
I am not sure this captures all the possible scenarios for the `Channel` sections. There's a few things we need to check:
- Where B1 = B2  or TF1 = TF2 if both of these are true then create the section
- Otherwise you need to check if B2 < Tolerance and TF2 < tolerance, in which case we can assume B1 = B2 and TF1 = TF2
- Else, the section is asymmetric and cannot be created